cluster/saltbase/salt/kube-apiserver/kube-apiserver.manifest

```diff
@@ -43,10 +43,12 @@
 
 {% set cert_file = "--tls_cert_file=/srv/kubernetes/server.cert" -%}
 {% set key_file = "--tls_private_key_file=/srv/kubernetes/server.key" -%}
+{% set client_ca_file = "--client_ca_file=/dev/null" -%}
```

Comment:
Sorry @derekwaynecarr. I was following the example of the token auth file since I thought the implementation on the server was identical.